styles/OpenLoanStyles.ts

```diff
@@ -1,5 +1,5 @@
 export const OpenLoanStyles = {
-  wrapper: 'w-screen flex justify-center items-center',
```

Comment:
Don't commit OpenLoan Styles, Roberto will handle it.

Reply:
Done